client/components/account-balances/strings.ts

```diff
@@ -24,5 +24,29 @@ export const fundLabelStrings = {
 	pending: __( 'Pending funds', 'woocommerce-payments' ),
 };
 
+export const learnMoreString = __( 'Learn more.', 'woocommerce-payments' );
+
+export const fundTooltipStrings = {
+	available: __(
+		'The amount of funds available to be deposited.',
+		'woocommerce-payments'
+	),
+	availableNegativeBalance: __(
+		'Learn more about why your account balance may be negative.',
+		'woocommerce-payments'
+	),
+	pending: __(
+		'The amount of funds still in the %d day pending period.',
```

Comment:
Do we want to get out of our ways to shows `days` when %d is larger than 1?

Reply:
Also, 
[delay days can be zero](https://stripe.com/docs/api/accounts/object#account_object-settings-payouts-schedule-delay_days). We should handle such case. I'm not sure when it's zero though, maybe when schedule is set to manual?

Reply:
> Do we want to get out of our ways to shows days when %d is larger than 1?

I think the current grammar is correct for any %d value, e.g.:

* the 1 day pending period
* the 2 day pending period
* the 0 day pending period

> We should handle such case. I'm not sure when it's zero though, maybe when schedule is set to manual?

I'm also not sure if this could occur in reality. We could hide the tooltip, or we could state "the 0 day pending period", which is technically correct.

Reply:
Even though the grammar is correct in english, I think we may still need to use a `_n` here. 

Some languages change the order of the words so that the grammar is no longer correct. eg in French the translation is "la période d'attente de 2 jours" which translates into "The waiting period is 2 days". By keeping this a simple `__()` it may complicate it for some translators to squeeze their language into a unplural format.

Reply:
Yep that's correct. :)